src/TestTraits/BehatProvidingTrait.php

```diff
@@ -24,8 +24,8 @@ trait BehatProvidingTrait  {
    * @return Behat\Gherkin\Node\FeatureNode
    *   A Behat feature.
    */
-  public function parseBehatFeature($featureString, $keywords = NULL) {
-    $lexer  = new Lexer($this->getBehatKeywords($keywords));
+  public static function parseBehatFeature($featureString, $keywords = NULL) {

```

Comment:
Why the need to move to static declarations? This will break backwards compatability badly for existing users.